bootcamp_module08/core/student_02.py

```diff
@@ -14,17 +14,5 @@ def count_substring(string, substring):
         The number of times `substring` occurs in `string`
 
     """
-    count = 0
 
-    string_length = len(string)
-    substring_length = len(substring)
-    n_subsequences = string_length - substring_length + 1
-
-    for i in range(n_subsequences):
-        left_bound = i
-        right_bound = i + substring_length
-        candidate_substring = string[left_bound:right_bound]
-        if candidate_substring == substring:
-            count += 1
-
-    return count
+    return string.lower().count(substring.lower())
```

Comment:
Nice python knowledge!

In an instance like this, there maybe a few considerations:
* Turning this one-liner into a few lines with informative variable names could greatly enhance readability
* Sometimes it may be better to change as little of the existing code as possible. Especially if large changes to the existing code are not strictly necessary for your new feature/bug, and you are not (one of) the core mainainter(s) of the package. That being said, its totally situational.